rear_camera_py/rear_camera/black_extractor.py

```diff
@@ -0,0 +1,46 @@
+"""黒抽出モジュール.
+
+リアカメラから取得した画像を黒白の2値画像データに変換する。
+@author kodama0720
+"""
+import cv2
+from typing import List
+import numpy as np
+import time
+
+
+class BlackExtractor:
+    """画像の黒抽出クラス."""
+
+    def extract_black(self, rear_camera_img: cv2.Mat) -> List[int]:
+        """画像から黒線を抽出する関数.
+
+        Args:
+            rear_camera_img (cv2.Mat): リアカメラ画像
+
+        Returns:
+            2値変換した画像データ: List[int]
+        """
+        lower = np.array([0, 0, 80])
+        upper = np.array([180, 245, 255])

```

Comment:
```suggestion
        MIN_WHITE_VALUE = 80             # 白と識別するための最低輝度
        MAX_WHITE_SATURATION=245 # 白と識別する最高彩度
        lower_hsv = np.array([0, 0, MIN_WHITE_VALUE])
        upper_hsv = np.array([180, MAX_WHITE_SATURATION, 255])
```
上記のような形(認識を間違えていたらすみません)かコメントで、2値判定の基準をわかりやすくしてほしいです。
特に調整次第で値が変わるものは明記してほしいです。

それと彩度の閾値が255ではなく245なのは意図的ですか？